src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py

```diff
@@ -176,7 +176,7 @@ def test_azure_vpn_server_config_basic_scenario(self, resource_group):
         ])
 
         self.cmd('network vpn-server-config delete -n {vserverconfig} -g {rg} -y')
-        with self.assertRaisesRegexp(SystemExit, '3'):
+        with self.assertRaisesRegex(SystemExit, '3'):
```

Comment:
Could you explain why we replace `assertRaisesRegexp` with `assertRaisesRegex`?

Reply:
Here's error of `assertRaisesRegexp`
```
         self.cmd('network vpn-server-config delete -n {vserverconfig} -g {rg} -y')
>       with self.assertRaisesRegexp(SystemExit, '3'):
E       AttributeError: 'AzureVWanVHubScenario' object has no attribute 'assertRaisesRegexp'. Did you mean: 'assertRaisesRegex'?

azure-cli-extensions\src\virtual-wan\azext_vwan\tests\latest\test_azure_vwan_vhub_commands.py:179: AttributeError
```
It's deprecated and renamed to `assertRaisesRegex()` for python3
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex